src/test/regress/expected/foreign_key_restriction_enforcement.out

```diff
@@ -290,10 +290,8 @@ BEGIN;
 DEBUG:  switching to sequential query execution mode
 DETAIL:  Reference relation "referece_table" is modified, which might lead to data inconsistencies or distributed deadlocks via parallel accesses to hash distributed relations due to foreign keys. Any parallel modification to those hash distributed relations in the same transaction can only be executed in sequential query execution mode
 	COPY on_update_fkey_table FROM STDIN WITH CSV;
-ERROR:  cannot execute parallel COPY on relation "on_update_fkey_table" after DML command on reference relation "referece_table" because there is a foreign key between them and "referece_table" has been modified in this transaction
-DETAIL:  COPY to a distributed table uses a separate set of connections which will not be able to see the uncommitted changes to the reference table.
-HINT:  Perform the COPY in a separate transaction.
-CONTEXT:  COPY on_update_fkey_table, line 2: "1002,99"
+ERROR:  insert or update on table "on_update_fkey_table_2380001" violates foreign key constraint "fkey_2383883"
```

Comment:
maybe adjust the test so that the data goes into the table via COPY? Because this currently looks slightly confusing for the readers. If you check the regression test file, we typically fail only if we cannot support the tx block.